cips/cip-034.md

```diff
@@ -1,36 +1,35 @@
 | cip | 34 |
 | - | - |
-| title | Lower Sampling and Pruning Window to 14 days |
-| description | Lower Sampling and Pruning Window to 14 days |
-| author | Callum Waters ([@cmwaters](https://github.com/cmwaters)), Nashqueue ([@Nashqueue](https://github.com/Nashqueue)) |
-| discussions-to | <https://forum.celestia.org/t/cip-lower-sampling-and-pruning-window-to-14-days/1965> |
-| status | Review |
-| type | Standards Track |
-| category | Data Availability |
-| created | 2025-03-17 |
+| title            | Set Pruning Window to 7 Days + 1 Hour                                                              |
+| description      | Sets the minimum data pruning window for nodes to 7 days + 1 hour             |
+| author           | Callum Waters ([@cmwaters](https://github.com/cmwaters)), Nashqueue ([@Nashqueue](https://github.com/Nashqueue)) |
+| discussions-to   | <https://forum.celestia.org/t/cip-lower-sampling-and-pruning-window-to-14-days/1965>                 |
+| status           | Review                                                                                             |
+| type             | Standards Track                                                                                    |
+| category         | Data Availability                                                                                  |
+| created          | 2025-03-17                                                                                         |
+| requires         | CIP-036                                                                                            |
 | supercedes | [CIP-4](./cip-004.md) |
 
 ## Abstract
 
-Lower the minimum sampling and pruning windows to 14 days and 14 days + 1 hour, respectively, to reduce storage costs in response to the network’s increasing throughput. This also ensures that the subjectively initialized header— which must be within the trusting period— can sample all headers within the sampling window without relying on backwards sync.
+This CIP proposes to set the minimum `PruningWindow` for Celestia nodes to 7 days + 1 hour (169 hours). This change is intended to complement CIP-036, which is expected to define a 7-day `SamplingWindow`. A pruning window of 7 days + 1 hour ensures that nodes can serve data for the entire sampling period. This adjustment aims to reduce storage requirements for node operators while maintaining data availability for light clients.
```

Comment:
[nit] can this clarify that it only applies to DA nodes?

```suggestion
This CIP proposes to set the minimum `PruningWindow` for Celestia DA nodes to 7 days + 1 hour (169 hours). This change is intended to complement CIP-036, which is expected to define a 7-day `SamplingWindow`. A pruning window of 7 days + 1 hour ensures that nodes can serve data for the entire sampling period. This adjustment aims to reduce storage requirements for node operators while maintaining data availability for light clients.
```

Reply:
Agree. Specifically Bridge DA nodes